### PR TITLE
fix: Hyperlinked words instead of entire textview

### LINKED ISSUE #1955: Whole text working as hyperlink
**Describe the bug**
In AboutUs fragment hyperlinks are highlighted in blue colour but link opens on clicking non highlighted text also

**Expected Behaviour**
Link should open only on clicking highlighted text

**Steps to reproduce it**
1.Open AboutUs fragment
2.Click on any text

**Smartphone Info:**

|               | |
|---------------|-|
|Device         |[ Moto G4 Plus]|
|Android Version| [Nougat 7.0]    |

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by atm1504**
Fixes #1955 

Changes: 
Replaced the older techniques used by using Span in AboutUsFragment
Modified the string file.

Screenshots for the change: 
clickable_word

### CODE REVIEW

**Comment by atm1504** [comment]
@iamareebjamal Sir, may I know the reason why ci build failed?

**Comment by immadisairaj** [comment]
> @iamareebjamal Sir, may I know the reason why ci build failed?

@atm1504 you can go through the CircleCI build log to find where the error is
https://circleci.com/gh/fossasia/susi_android/4568
this is the link to this PR

**Comment by atm1504** [comment]
@iamareebjamal I tried reformatting the code, but the also circleci/ci build failed. Please help me how to solve it

**Comment by iamareebjamal** [comment]
Did you read the reason of failure?

**Comment by atm1504** [comment]
Yes sir. I corrected that. But after that also, the build failed

**Comment by iamareebjamal** [comment]
Impossible. Read build logs again

**Comment by atm1504** [comment]
> Impossible. Read build logs again

Ok sir. Let me try again

**Comment by atm1504** [comment]
@iamareebjamal can u please help. I am unable to fix it

**Comment by iamareebjamal** [comment]
Run spotless locally and fix the errors

**Comment by atm1504** [comment]
> Run spotless locally and fix the errors

Can you please tell me the way how to do that?

**Comment by iamareebjamal** [comment]
https://github.com/diffplug/spotless

**Comment by atm1504** [comment]
@iamareebjamal please review it now. Thank you for your help to solve spotless errors

File: app/src/main/java/org/fossasia/susi/ai/skills/aboutus/AboutUsFragment.kt [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-import android.text.Html
+import android.text.SpannableString
 import android.text.Spanned
+import android.text.method.LinkMovementMethod
+import android.text.style.ClickableSpan
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.Toast
+import kotlinx.android.synthetic.*
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/skills/aboutus/AboutUsFragment.kt]: Don't use wildcard imports

**Comment by atm1504** [comment]
@iamareebjamal please review it.

**Comment by iamareebjamal** [comment]
I think using anchors to display links is better than using manual spans

**Comment by atm1504** [comment]
Sir, if I use anchor then it will hyperlink the entire text view instead of hyperlinking individual words.

**Comment by iamareebjamal** [comment]
Not true. In over 50 apps I have created, it's not the case

**Comment by atm1504** [comment]
@iamareebjamal sir, there is some problem with the anchor tag. You can also see that initially it was using anchor tag itself, but it did not work. I tried today in different ways, but it didn't work. Instead of manual spanning i can make the span programmaticaly. This will make the code more dynamic. Please provide some suggestions.

**Comment by iamareebjamal** [comment]
Every app I know uses anchor tags. Find out why it isn't working

**Comment by atm1504** [comment]
@iamareebjamal sir I have tried again, but it failed repeatedly. I am failing to understand, why anchor tag is not working here. Can you please help?

**Comment by iamareebjamal** [comment]
@atm1504 Please create a sample app with a textview and use Html.fromHtml in it and see that anchor tags work

**Comment by atm1504** [comment]
@iamareebjamal sir, I have tested it in a sample app and found that it is working properly if we write the string in the .kt file. But it is not working as soon as we use string from the string file. This is because the method in which the strings were written in the string file was wrong. I need to reformat the code, in order to run this. I will soon make the changes and make the pr.
